docs(core): add nx console troubshooting guide (#19715)
nrwl/nx: docs/shared/recipes/console-troubleshooting.md

# Nx Console Troubleshooting

## VSCode + nvm Issues

> VSCode loads a version of Node when it starts. It can use versions set via [`nvm`](https://github.com/nvm-sh/nvm) but there's some caveats.

Reviewer: ```suggestion
VSCode loads a version of Node when it starts. It can use versions set via [`nvm`](https://github.com/nvm-sh/nvm) but there are some caveats.
```

> - If you've installed Node outside of `nvm` (for example using their installer or `brew` on Mac), VSCode will always use that version. You can check by running `nvm list` and looking for a `system` alias. To enable VSCode to pick up your `nvm` version, make sure to uninstall it.

Reviewer: ```suggestion
- If you've installed Node outside of `nvm` (for example using the Node installer or `brew` on Mac), VSCode will always use that version. You can check by running `nvm list` and looking for a `system` alias. To enable VSCode to pick up your `nvm` version, make sure to uninstall the version of Node that was installed outside of `nvm`.
```

> - VSCode will load the `default` alias from `nvm` at startup. You can set it by running `nvm alias default [version]`. To make sure VSCode can load the correct value needs to be set in your OS' default terminal. Setting it in a VSCode-integrated terminal won't persist after it's closed. Similarly, setting it in a third-party app like iTerm won't influence VSCode by default.

Reviewer: ```suggestion
- VSCode will load the `default` alias from `nvm` at startup. You can set it by running `nvm alias default [version]`. The `default` alias needs to be set in your OS' default terminal for VSCode to pick it up. Setting it in a VSCode-integrated terminal won't persist after it's closed. Similarly, setting it in a third-party app like iTerm won't influence VSCode by default.
```
